### PR TITLE
bugfix/SOF-3813

### PR DESCRIPTION
**Pull request by mohammadidinani**

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -20,43 +20,41 @@ export function getElectronegativity(symbol) {
...
- * Determines whether elements are bonded. Elements are bonded if the distance is equal or less than
- *  - the bond length, if bond length exists, or
- *  - the sum of covalent radii times the chemical connectivity factor (http://www.xcrysden.org/doc/modify.html).
+ * Returns bond information for the given elements.
  */
-export function areElementsBonded(element1, element2, distance, order = undefined, chemicalConnectivityFactor = 1.05) {
-    const allBonds = getElementsBondData(element1, element2, order, chemicalConnectivityFactor);
-    return Boolean(allBonds.find(b => b.length.value && b.length.value >= distance));
+export function getElementsBondsData(element1, element2, order = undefined) {
+    const defaultElementsBondsData = getDefaultElementsBondsData(element1, element2, order);
```
Comment (timurbazhirov) [inline src/js/index.js]: defaultElementsBondsDataEntry

File: src/js/index.js [hunk 2]
```diff
@@ -20,43 +20,41 @@ export function getElectronegativity(symbol) {
     return config ? config["pauling_negativity"] : 0;  // return zero if value cannot be accessed by symbol
 }
 
+export function filterBondsDataByElementsAndOrder(bondsData, element1, element2, order = undefined) {
```
Comment (timurbazhirov) [inline src/js/index.js]: Let's have the docstring above and properly format the ones below too - with @param, @return etc

Comment (mohammadidinani) [reply src/js/index.js]: Done.

File: src/js/index.js (hunk 1 above)
Comment (mohammadidinani) [reply src/js/index.js]: Done.

**Pull request merged**